### DISCUSSION TITLE
City not found and form errors

### DISCUSSION THREAD

**Discussion by matheusbn** [joemasilotti/railsdevs.com#903] [Q&A]
Hey, I'm trying to build my profile, but when I submit the form I get the following errors:
![image](https://github.com/joemasilotti/railsdevs.com/assets/12435310/a459fcb9-c25f-4ae9-9a5c-91bdee1fce12)

The city name is correct and I don't see any field for location timezone or UTC offset.
I have already disabled the adblock.

**Comment by joemasilotti** [comment]
Thanks for the report! This is now fixed.
